blockdata/blockdata.go

```diff
@@ -119,14 +136,22 @@ func (t *Collector) CollectAPITypes(hash *chainhash.Hash) (*apitypes.BlockDataBa
 // the block data required by Collect() that is specific to the block with the
 // given hash.
 func (t *Collector) CollectBlockInfo(hash *chainhash.Hash) (*apitypes.BlockDataBasic,
-	*dcrjson.FeeInfoBlock, *dcrjson.GetBlockHeaderVerboseResult, error) {
+	*dcrjson.FeeInfoBlock, *dcrjson.GetBlockHeaderVerboseResult, *apitypes.BlockExplorerExtraInfo, error) {
 	msgBlock, err := t.dcrdChainSvr.GetBlock(hash)
 	if err != nil {
-		return nil, nil, nil, err
+		return nil, nil, nil, nil, err
 	}
 	height := msgBlock.Header.Height
 	block := dcrutil.NewBlock(msgBlock)
-
+	txLen := len(block.Transactions())
+	coinSupply, _ := t.dcrdChainSvr.GetCoinSupply()
+	nbSubsidy, _ := t.dcrdChainSvr.GetBlockSubsidy(int64(msgBlock.Header.Height)+1, 1)
```

Comment:
please comment about the 1 voter thing, assuming the idea is to get a per/voter PoS subsidy, rather than a block total

Reply:
Since I can't really tell how many votes would be in the next block(it's usually 5 but sometimes 4), just decided to use 1 but I can use 5 if you want

Reply:
Right, so I think it only makes sense to show the reward for a single voter (and indicating somewhere that it is per vote) or for all 5 which would imply the max possible subsidy.  I like 1 if we label it clearly.